src/mongo/db/ldap/ldap_manager_impl.cpp

```diff
@@ -170,23 +191,58 @@ int cb_add(LDAP *ld, Sockbuf *sb, LDAPURLDesc *srv, struct sockaddr *addr,
     int fd = -1;
     ldap_get_option(ld, LDAP_OPT_DESC, &fd);
     LOG(2) << "LDAP connect callback; file descriptor: " << fd;
-    static_cast<LDAPManagerImpl::ConnectionPoller*>(ctx->lc_arg)->start_poll(fd);
+    //static_cast<LDAPManagerImpl::ConnectionPoller*>(ctx->lc_arg)->start_poll(fd);
```

Comment:
it seems this was the only place where start_poll was called

Reply:
I committed this change by accident, I only disabled it for testing to see what it does. Readded this call.